staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -878,6 +899,10 @@ func (qs *queueSet) finishRequestAndDispatchAsMuchAsPossible(req *request) bool
 // previously dispatched request has completed it's service.  This
 // callback updates important state in the queueSet
 func (qs *queueSet) finishRequestLocked(r *request) {
+	if qs.seatDemandIntegrator == nil {
+		klog.ErrorS(nil, "nil seatDemandIntegrator", "QueueSet", qs.qCfg.Name)
+		panic(qs)
```

Comment:
I'm not a fan of it...
Can we instead move this to the point where queueSet is created (which effectively is only queueSetCompleter::Complete?

Reply:
Removed, it was just debugging for something that should never happen.